src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs

```diff
@@ -35,20 +35,18 @@ public static partial class ImageSourceExtensions
 				return null;
 			}
 
-			UIGraphics.BeginImageContextWithOptions(imagesize, false, scale);
-			var ctx = new NSStringDrawingContext();
-
-			var boundingRect = attString.GetBoundingRect(imagesize, 0, ctx);
-			attString.DrawString(new CGRect(
-				imagesize.Width / 2 - boundingRect.Size.Width / 2,
-				imagesize.Height / 2 - boundingRect.Size.Height / 2,
-				imagesize.Width,
-				imagesize.Height));
-
-			var image = UIGraphics.GetImageFromCurrentImageContext();
-			UIGraphics.EndImageContext();
-
-			return image.ImageWithRenderingMode(UIImageRenderingMode.Automatic);
+			var renderer = new UIGraphicsImageRenderer(imagesize);
```

Comment:
For example, here you switched directly to the new way without keeping the old way. I think this is better? Thoughts?